src/EmailServiceClientException.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+namespace Sil\Idp\IdBroker\Client;
+
+class EmailServiceClientException extends \Exception
```

Comment:
Should we consider renaming this since it technically isn't the email service anymore?